### DISCUSSION TITLE
Can I use HTTP URL in `localePath` ?

### DISCUSSION THREAD

**Discussion by jtjisgod** [i18next/next-i18next#1455] [General]
Can I use HTTP url in `localePath`? 
I will provide formed data through Django restful api.

```javascript
const path = require("path");

module.exports = {
  i18n: {
    locales: ["tr", "en"],
    defaultLocale: "tr",
    fallbackLng: "en",
    ns: ["common"],
  },
  localePath: path.resolve("http://api.example.com/locales/"), // HERE
};
```

**Comment by isaachinman** [comment]
Here's [one example](https://github.com/locize/next-i18next-locize) of an http backend. If you want to use your own API, you'll need to write your own backend plugin.

**Reply by jtjisgod** [reply]
Thank you!
